src/main/java/seedu/tinner/model/company/FavouriteStatus.java

```diff
@@ -10,7 +10,7 @@
 
     public static final String MESSAGE_CONSTRAINTS =
             "Favourite should only contain numbers, and it should be at least 3 digits long";
-    public final Boolean isFavourited;
+    public final Boolean value;
```

Comment:
I think reverting to use isFavourited may be more appropriate to adhere to the naming of Boolean attributes.

Reply:
This change was made to better adhere the format of the other company fields; `Phone`, `Email`, and `Address` all seem to use `value` to denote the content they store. Can definitely see where you're coming from though, I'll try to get some more clarification on this and revert as necessary.